added thre test for womens clothes, but get no respons from endpoint,…

mhuggare opened the pull request:
* test for endpoint status 200
* est for a faulty endpoint to get status 404
* test to see that one product is as part of the respons. but get nothing back. need to update test later if i get something back from the endpoint

mhuggare committed:
added thre test for womens clothes, but get no respons from endpoint, so need to update test later

korsbacken commented:
Did you create your test cases in the Cucumber test runner class intentionally? I think this file "should" be empty and just reference the cucumber feature file 🤔 I created a class file called ApiTests.java where I put my api tests in. And they ran fine with Github Actions.

mhuggare committed:
Merge branch 'master' into Mia

mhuggare commented:
no i started to add in cucumber class, but then i saw that you created a new branch and most have forgotten the things i had added to cucumber.
i deleted the cucumer file stuff and now it seems my conflict is gone...

korsbacken reviewed (approved):
Approved.
